File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/ProductSpecifications/IProductSpecificationData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.ProductSpecifications
{
    public interface IProductSpecificationData : CoreObject.IObjectID, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IProductSpecificationValue, IProductSpecificationSpecification
    {
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductSpecifications/IProductSpecificationData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IProductSpecificationData extends IProductSpecificationSpecification,IProductSpecificationValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
